src/main/webapp/less/portlet-grid.less

```diff
@@ -0,0 +1,142 @@
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*---------------------------------------------------
+    PORTLET GRID Mixin   | version: 0.7
+  ---------------------------------------------------
+    A mixin to reflow Bootstrap Classes and to create
+    a "css breackpoint" within the portlet Grid in 
+    Desktop Multi columns Layout
+  ---------------------------------------------------*/
+
+
+.grid( @name; @columns; @col-md-breackpoint; @index: 1) when (@index =< @columns) {
+    .grid(@name; @columns; @col-md-breackpoint; (@index + 1));
+
+     & when ( @container-fluid  = 0 ) {
+        .row {
+           margin-left: -20px;
+           margin-right: 0;
+        }
+     }
+
+    .@{name}-@{index} {
+
+      & when ( @container-fluid  = 1 ) {
+
+        /* When the portlet have ".container-fluid" Class */
+
+            & when ( @index < @portletbreackpoint ) {
+                width: ( percentage(@index / @portletbreackpoint ));
+                float: left;
+            }
+
+
+            & when ( @index >= @portletbreackpoint ) {
+               width: ( percentage(1));
+               float: left;
+               clear:both;
+            }
+            
+        }
+
+      & when ( @container-fluid  = 0 ) {
+
+        /* Work In Progress (WIP) for .container Bootstrap Class (not .container-fluid) */
+
+            & when ( @index < @portletbreackpoint ) {
+
+               & when ( percentage(@index / @columns * @columns / @portletbreackpoint / 12 * @col-md-breackpoint ) < 50% ) {
+                  width: ( percentage(@index / @columns * @columns / (@portletbreackpoint) / 12 * @col-md-breackpoint ) - percentage(0.026) );
+                  float: left;
+               }
+
+               & when ( percentage(@index / @columns * @columns / @portletbreackpoint / 12 * @col-md-breackpoint ) < 32% ) {
+                  width: ( percentage(@index / @columns * @columns / (@portletbreackpoint) / 12 * @col-md-breackpoint ) - percentage(0.02) );
+                  float: left;
+               }
+
+               & when ( percentage(@index / @columns * @columns / @portletbreackpoint / 12 * @col-md-breackpoint ) < 24% ) {
+                  width: ( percentage(@index / @columns * @columns / (@portletbreackpoint) / 12 * @col-md-breackpoint ) - percentage(0.013));
+                  float: left;
+               }
+
+            }
+
+            & when ( @index >= @portletbreackpoint ) {
+               width: ( percentage(0.5) - percentage(@portletbreackpoint / 3 * 2 / 100) );
```

Comment:
Do not pay attention this is WIP. it is a attempt to recalculate a Fixed (not fluid) grid inside the fluid grid of respondr...
in case of someone has the bad practice to use Bootstrap "container" Class instead of "container-fluid...
I warn every one [here](https://github.com/cousquer/FeedbackPortlet/blob/44a6f3859f60a5c093fe0a5e227f29db6eae9728/src/main/webapp/less/variables.less#L27)...